tar/multitape/multitape_delete.c

```diff
@@ -76,6 +77,7 @@ deletetape(TAPE_D * d, uint64_t machinenum, const char * cachedir,
 	int lockfd;
 	uint8_t lastseq[32];
 	uint8_t seqnum[32];
+	int rc = -1;	/* Assume an error will occur; will change if needed. */
```

Comment:
I'd propose "Assume a non-archive-not-found error; will change if needed", but that's more than 80 characters.